README.md

```diff
@@ -438,3 +438,25 @@ npm run dev -- <command> <options>
 ```
 npm run test
 ```
+
+## Performance testing 
+
+To run performance testing for OA functionality
+
+### Wrap
+
+Monitor the response time for batched documents wrapping. 
+
+The Default command will test with 2 batched documents in 1 iteration.
+```
+npm run benchmark
+```
+
+The number of documents and iteration can be modified using these options.
+- First number : Number of document for batched wrapping
+- Second number : Number of iteration.
```

Comment:
I'm not sure Number of Iteration is very clear, can be more precise ?

Reply:
Done. Change to "Number of performance test iteration to achieve higher accuracy"